tchannel-core/src/main/java/com/uber/tchannel/frames/ErrorFrame.java

```diff
@@ -76,4 +80,19 @@ public String toString() {
         );
     }
 
+    public static void sendError(ErrorType type, String message, RawRequest request, ChannelHandlerContext ctx) {
```

Comment:
It is a common utility whenever error responses need to be sent.

ref: https://github.com/uber/tchannel-java/pull/89/files#diff-0c30656be54ed88283a28294e0fa1555R94